packages/playground/src/portal/dao_view.vue

```diff
@@ -89,7 +89,7 @@
                   <v-row v-if="expired(proposal.end)" class="">
                     <div :style="{ width: `${proposal.ayesProgress}%` }">
                       <v-progress-linear
-                        :height="20"
+                        :height="24"
                         color="#1AA18F"
```

Comment:
We can use primary color instead of the hex code here.

Reply:
Fixed